mondrian/src/main/java/mondrian/calc/impl/ArrayTupleList.java

```diff
@@ -24,6 +25,7 @@
 {
     private transient Member[] objectData;
     private int size;
+    private static final int cjMaxSize = MondrianProperties.instance().ResultLimit.get();
```

Comment:
@mkambol Should _**static**_ be removed?  It is possible to re-load the Mondrian properties at run-time.  I checked all current uses of this ResultLimit property and nowhere else saves it into a static location.

Reply:
Good catch!